libs/features/pages/content-development/src/lib/content-dev-hero-section/content-dev-hero-section.component.scss

```diff
@@ -0,0 +1,95 @@
+/* Styling for the content-main container */
+.content-main {
+top: 0px;
+left: 0px;
+width: 1920px;
+height: 1082px;
+opacity: 1;
+font-size: medium;
+margin-left: 100px;
+}
+
+/* Styling for the hero-content */
+.hero-content {
+    text-align: start;
+    margin: 100px; /* Add margin as needed */
+    position: relative; /* Needed for z-index to work */
+    margin-left: 300px;
+    width: 100px;
+    padding: 40px;
+  }
+/* Styling for the text-section container */
+.text-section {
+    width: 1200px;
+    font-size: medium;
+    /* Add additional styles for the text section as needed */
+  }
+
+/* Styling for the rectangles container */
+.rectangles {
+    position: absolute; /* Position rectangles absolutely within content-main */
+    top: 0;
+    left: 0;
+    display: flex;
+    justify-content: space-between;
+    z-index: -1; /* Place rectangles behind content */
+    margin-right: 400px;
+
+  }
+/* Styling for individual feat-one, feat-two, and feat-three divs */
+.feat-one {
+    top: 737px;
+    left: 291px;
+    width: 462px;
+    height: 461px;
+    /**UI Properties*/
+    background: transparent linear-gradient(180deg, #F2CF73 0%, #F2CF7300 100%) 0% 0% no-repeat padding-box;
```

Comment:
Use CSS variables specified in elements/theming/_variables.scss, to specify colors for the linear gradient

Reply:
*resolved